Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM golang:1.9-alpine AS build
```

Comment:
Why not using the latest go for the build? For instant, 1.11.5?

Reply:
Sure, I was just trying to be consistent with `.travis.yml`. Will bump to 1.11.